novawallet/Modules/WalletsList/Common/WalletsListViewModelFactory.swift

```diff
@@ -100,24 +100,80 @@ extension WalletsListViewModelFactory: WalletsListViewModelFactoryProtocol {
         let totalValue = formatPrice(amount: totalValueDecimal, locale: locale)
 
         let optIcon = wallet.info.walletIdenticonData().flatMap { try? iconGenerator.generateFromAccountId($0) }
-        let iconViewModel = optIcon.map { DrawableIconViewModel(icon: $0) }
+        let iconViewModel = optIcon.map { IdentifiableDrawableIconViewModel(
+            .init(icon: $0),
+            identifier: wallet.info.metaId
+        ) }
+
+        let walletViewModel = WalletView.ViewModel(
+            icon: iconViewModel,
+            networkIcon: nil,
+            name: wallet.info.name,
+            subtitle: totalValue,
+            subtitleDetailsIcon: nil,
+            subtitleDetails: nil,
+            marked: false
+        )
+
+        return WalletsListViewModel(
+            identifier: wallet.identifier,
+            walletViewModel: walletViewModel,
+            isSelected: isSelected(wallet: wallet)
+        )
+    }
+
+    func createProxyItemViewModel(
+        for wallet: ManagedMetaAccountModel,
+        wallets: [ManagedMetaAccountModel],
+        chains: [ChainModel.Id: ChainModel],
+        locale: Locale
+    ) -> WalletsListViewModel? {
+        guard let chainAccount = wallet.info.chainAccounts.first(where: { $0.proxy != nil }),
+              let proxy = chainAccount.proxy,
+              let proxyWallet = wallets.first(where: { $0.info.has(
+                  accountId: proxy.accountId,
+                  chainId: chainAccount.chainId
+              ) && $0.info.type != .proxied })
```

Comment:
I believe we display proxieds in the section. Why do need this check: != .proxied? As I understand ```wallet``` is already proxied.

Reply:
we are looking for proxy for this proxied-wallet